youtube_dl/extractor/gaskrank.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from re import search
+from re import findall
+from .common import InfoExtractor
+
+
+class GaskrankIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?gaskrank\.tv/tv(?:/[^/]+)+/(?P<id>[^/]+).htm'
+    _TEST = {
+        'url': 'http://www.gaskrank.tv/tv/motorrad-fun/strike-einparken-durch-anfaenger-crash-mit-groesserem-flurschaden.htm',
+        'md5': '1ae88dbac97887d85ebd1157a95fc4f9',
+        'info_dict': {
+            'id': '201601/26955',
+            'ext': 'mp4',
+            'title': 'Strike! Einparken können nur Männer - Flurschaden hält sich in Grenzen *lol*',
+            'thumbnail': r're:^https?://.*\.jpg$',
+        }
+    }
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        video_id = self._search_regex(r'https?://movies.gaskrank.tv/([^-]*?).mp4', webpage, 'video id')
+        playlist = self._search_regex(r'playlist:\s*\[\s*{\s*([\s\S]*(?!}\s*]))', webpage, 'video id')
+        entries = findall(r'[0-9]+:\s*{[\s\S]*?}', playlist)
```

Comment:
Fixed, but it didn't go smooth.

This js-code
`		        var myPlayer = projekktor('#playera', {
		addplugins: ['logo','ima'],
		//platforms: ['browser','flash'],
		//platforms: ['browser', 'android', 'ios', 'native', 'flash', 'vlc'],
		platforms: ['browser', 'android', 'ios', 'native', 'flash'],
`
wasn't converted properly by js_to_json since the commented lines are not removed.

Also, the js-code contains a trailing comma in a dict which json.loads does not tolerate.

But anyhow, I worked around both.